src/components/POIForm.component.js

```diff
@@ -1,9 +1,215 @@
 import React, { Component } from 'react'
+import {
+  FormGroup,
+  FormControl,
+  ControlLabel,
+  Button,
+  Checkbox,
+  Form,
+  Col,
+  PageHeader
+} from 'react-bootstrap'
+import DatePicker from 'react-datepicker'
+import moment from 'moment'
+import 'react-datepicker/dist/react-datepicker.css'
 
 class POIForm extends Component {
+  constructor(props) {
+    super(props)
+    this.onDateSelected = this.onDateSelected.bind(this)
+    this.state = {
+      startDate: moment(),
+      name: '',
+      description: '',
+      selectedStories: [],
```

Comment:
Since we're not doing anything with this right now, and since we don't know exactly how this will be used when selecting stories, let's leave this out for now